src/reader/widgets/ReaderWidget.vue

```diff
@@ -103,6 +117,21 @@
                 endCursor
               }
             }
+            imageAnnotations(reference: "${this.urn}") {
```

Comment:
I've created ![](https://github.trello.services/images/mini-trello-icon.png)[Use conditional query for passage data in ReaderWidget](https://trello.com/c/PNFozRir) for a future iteration.  Not worth blocking, but over time, let's hew to the customization of GraphQL queries so we only query for the data that we need.